var/spack/repos/builtin/packages/racon/package.py

```diff
@@ -25,10 +29,8 @@ class Racon(CMakePackage):
     conflicts('%gcc@:4.7')
     conflicts('%clang@:3.1')
 
+    patch('aarch64.patch', when='target=aarch64:')
+
     def cmake_args(self):
         args = ['-Dracon_build_wrapper=ON']
         return args
-
-    def install(self, spec, prefix):
```

Comment:
Does this mean the old versions no longer build?

Reply:
I tested the old version, it looks work well without this install() function.
But new version will build fail with this function.